utils/compare_minions.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import salt.config
+import salt.loader
+
+import json
+import json_delta
+
+try:
+    from colorama import Fore, Back, Style, init
```

Comment:
Unused imports.

offtop: I prefer to write import like this:
```
from colorama import (
    Fore, 
    Back, 
    Style, 
    init,
)
```
— this will bring much nicer git diffs in future

but in python community it's sometimes recommended to write import like this:
```
from colorama import Back
from colorama import Fore
from colorama import Style
from colorama import init
```